cli/internal/helm/action.go

```diff
@@ -9,21 +9,28 @@ package helm
 import (
 	"context"
 	"fmt"
+	"path/filepath"
 	"time"
 
 	"github.com/edgelesssys/constellation/v2/internal/constants"
+	"github.com/edgelesssys/constellation/v2/internal/file"
 
 	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chartutil"
 	"helm.sh/helm/v3/pkg/cli"
 )
 
 const (
 	// timeout is the maximum time given per helm action.
-	timeout = 10 * time.Minute
+	timeout         = 10 * time.Minute
+	installChartDir = "helm-install"
+	upgradeChartDir = "helm-upgrade"
 )
 
 type applyAction interface {
 	Apply(context.Context) error
+	SaveChart(upgradeDir string, fileHandler file.Handler) error
```

Comment:
```suggestion
	SaveChart(fileHandler file.Handler) error
```

What was the original motivation for differentiating between `helm-install` and `helm-upgrade`? 
The content of both folders should be the same, irrespective of `init` or `upgrade apply`. Saving the files to different folders introduces a (admittedly small) difference between `init` and `upgrade`. I think this isn't necessary?

Reply:
They are not saved to different directories for `init` or `upgrade`, but depending on if it is a new chart that has not been installed yet, or if it is a chart that will be upgrade.
A `constellation upgrade apply` can write both directories.
My reasoning for this was to make it more clear for end users what charts are updated by an upgrade and what charts have to be newly installed.

Reply:
We can tell users to just use `helm upgrade --install`? We will have to resolve the difference between `init` and `upgrade` very soon anyways, so I would prefer to do it now.

Reply:
> We can tell users to just use `helm upgrade --install`?

That would be possible.

> We will have to resolve the difference between `init` and `upgrade`

Again, this has nothing to do with init and upgrade, this is purely to differentiate if a chart will be installed as a new chart (because it does not yet exist in the cluster) or if it has to be upgraded. The `helm-install` directory may also exist on `constellation upgrade apply` if we added new charts to the release.
My reasoning for keeping them in separate folder was to highlight what is installed new and what is upgraded, which could make trouble shooting easier.